### PR TITLE
updates README.md and adds tap-html

### PR DESCRIPTION
**Pull request by gabrielcsapo**
😄 worked on tap-html, thought it might be a cool addition to the reporters section!

> the readme.markdown was giving me OCD a hard time, I can revert the name if the change was a problem.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,10 +15,10 @@ var test = require('tape');
 
 test('timing test', function (t) {
     t.plan(2);
-
```
Comment (ljharb) [inline README.md]: please revert all the unrelated whitespace changes.

**Comment by gabrielcsapo** [comment]
@ljharb changed the name of the file back 😄

**Comment by gabrielcsapo** [comment]
Thank you!
